src/controllers/WhoWasIllController.ts

```diff
@@ -0,0 +1,18 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller, httpGet } from 'inversify-express-utils';
+
+import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { Feature } from 'app/utils/Feature';
+import { WHO_WAS_ILL_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'illness/who-was-ill';
```

Comment:
Similar to before, the template is not the same thing as the URL. this should be the name of the file in the view directory.

```suggestion
const template = 'who-was-ill';
```

Reply:
Similiar to before - it is :)

Reply:
oh fancy